docs/content/users/extend/customizing-images.md

```diff
@@ -99,13 +99,20 @@ For more complex requirements, you can add:
 
 These files’ content will be inserted into the constructed Dockerfile for each image. They are inserted *after* most of the rest of the things that are done to build the image, and are done in alphabetical order, so `Dockerfile` is inserted first, followed by `Dockerfile.*` in alphabetical order.
 
-For certain use cases, you might need to add directives very early on the Dockerfile like proxy settings or SSL termination. You can use `pre.` variants for this that are inserted *before* everything else:
+For certain use cases, you might need to add directives very early on the Dockerfile like proxy settings or SSL termination. You can use `pre.` variants for this that are inserted *before* what DDEV adds to build the image:
 
 * `.ddev/web-build/pre.Dockerfile.*`
 * `.ddev/web-build/pre.Dockerfile`
 * `.ddev/db-build/pre.Dockerfile.*`
 * `.ddev/db-build/pre.Dockerfile`
 
+Finally, to support [Multi-stage builds](https://docs.docker.com/build/building/multi-stage/), you can use `stage.` variants that are inserted *before* everything else, *on top* of the generated Dockerfile.
```

Comment:
Can we also give a small example of how to use it right here inside DDEV?

I think newbies won't understand what they can do, even after reading Docker docs, e.g add a simple thing inside the container and a note about the presence of a second `Dockerfile` with `COPY --FROM ,,,`.

Thanks.

Reply:
@stasadev done!